projects/core/components/root/root.template.html

```diff
@@ -3,7 +3,7 @@
     class="scrollbar"
 ></tui-scroll-controls>
 <tui-svg-defs-host></tui-svg-defs-host>
-<tui-portal-host>
+<tui-dropdown-host>
```

Comment:
why rename to `dropdown` while there may be notifications components?

Reply:
the host is used only for dropdowns and sidebar  (sidebar will be deprecated and new sidebar will be on its own host)